effects/zio1/src/test/scala/sttp/client3/impl/zio/ZioRuntimeUtils.scala

```diff
@@ -0,0 +1,20 @@
+package sttp.client3.impl.zio
+
+import scala.concurrent.Future
+
+import zio.{Runtime, Task, ZEnv, ZIO}
+
+import sttp.client3.testing.ConvertToFuture
+
+trait ZioRuntimeUtils {
+
+  val runtime: Runtime[ZEnv] = Runtime.default
+
+  val convertZioTaskToFuture: ConvertToFuture[Task] = new ConvertToFuture[Task] {
```

Comment:
You are sure that, functionally it will remain the same ?

Reply:
All tests pass. It is just the conversion to a future in order to use scala-test, right? 
I have also used this on other projects where I still rely on Scala-Test for testing but the tests are all written in ZIO.

Reply:
if we look at the ZIO runToFuture implementation it is sort of the same:
https://github.com/zio/zio/blob/be586a263393af3962e9128d2336e1818f2ae6ef/core/shared/src/main/scala/zio/Runtime.scala#L159